generated/proto/persist_fs_commitlog.proto

```diff
@@ -0,0 +1,25 @@
+syntax = "proto3";
```

Comment:
yeah that'll lead to more conflicts, let's leave it as is (and maybe with a TODO) for now and fix it up once the two diffs are landed.